securedrop_client/gui/widgets.py

```diff
@@ -667,6 +668,9 @@ class SourceList(QListWidget):
     """
 
     CSS = '''
+    #sourcelist {
+        border: none;
+    }
```

Comment:
yeah, the idea is to use a line widget by itself instead of borders. it came up while you were out that there are no borders in the spec

Reply:
also see https://github.com/freedomofpress/securedrop-client/issues/511 for an example of where we need a line (not a border)

Reply:
hmm, maybe i'm missing something: what is the difference between CSS border-right and a line? (i.e. i'm wondering why make another widget that we have to cover with the selected box when we can just use CSS and add the styling there)

Reply:
i see no difference, as long as we can add shadowing. all the lines should have shadowing, right @ninavizz? i might have misunderstood this in the last meeting. but yeah you make a good point.

Reply:
> i'm wondering why make another widget that we have to cover with the selected box when we can just use CSS and add the styling there

this is trickier than it first appears. the source widget needs to become 100% transparent when selected so we can't just cover up the line. we could ditch the transparency solution and just apply a background that is _close_ to the background of the conversation view (i personally prefer this). i think how we draw this line will make a difference in the css/ qt tricks we use to partially cover it up.this can be addressed in #511.

Reply:
also we could add the line there for now using border-right. i like that idea because it'll look closer to the overall design.